src/pages/lobby.tsx

```diff
@@ -16,25 +19,42 @@ const Lobby: FC = () => {
   api.lobby.onQueueUpdate.useSubscription(undefined, {
     async onStarted() {
       try {
+        console.log("lobby:22");
```

Comment:
Remove
Also log in line 29, 37 & 56

Reply:
yeah that's my bad, the commit to clean that hadn't gone through, should be good now